typescript2/app-stdlib-matrix/src/index.css

```
@@ -0,0 +1,80 @@
+@import "tailwindcss";
+
+/* The two languages' colours. Everything else is Tailwind's palette; these
+   two are data, not decoration, so they get names. */
+@theme {
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@theme`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)